dashboard/db/migrate/20160315215912_create_paired_user_levels.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePairedUserLevels < ActiveRecord::Migration
+  def change
+    create_table :paired_user_levels do |t|
```

Comment:
What's the PK on this table? or rather, what is it indexed on? I'm guessing driver/nav